.github/workflows/test_url.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+for i in range(0,6):
+    command = "curl 'https://admin:admin@localhost:9200' -H 'Content-Type:application/json' --insecure -v"
+    result = os.system(command)
+    if result !=0:
+        os.system("sleep 30s")
+    else:
+        os.system("sleep 30s")
```

Comment:
Why are we adding sleep here

Reply:
Another 30 seconds are required for the OpenSearch domain to become operational.